unfetter-discover/unfetter-ui
# Ngrx comments

This requires `ngrxComments` branch for store as well.

Testing this requires access to two github accounts.

Notification Test plan:

1. Create an analytic with user A
2. Comment on the new analytic with user B
3. Ensure Notification displays for user A

Live Comment Test plan:

1. Stay in the analytic hub with both users
2. Ensure that comments between the two are updated in live time
